examples/example2.cpp

```diff
@@ -102,7 +102,25 @@ int main() {
         // } else {
         //     std::cout << "failed to place order\n";
         // };
-        //! [placing an order]
+        //! [placing an iceberg order]
+        /* std::string orderId = Kite.placeOrder(kc::placeOrderParams()
+                                                  .Symbol("SBIN")
+                                                  .Exchange("NSE")
+                                                  .TransactionType("BUY")
+                                                  .OrderType("LIMIT")
+                                                  .Price(580.0)
+                                                  .Quantity(2000)
+                                                  .Product("MIS")
+                                                  .IcebergQuantity(1000)
+                                                  .IcebergLegs(2)
+                                                  .ValidityTtl(2)
+                                                  .Variety(kc::VARIETY_ICEBERG)
+                                                  .Validity(kc::VALIDITY_TTL));
+        if (!orderId.empty()) {
+            std::cout << "order ID: " << orderId << "\n";
+        } else {
+            std::cout << "failed to place iceberg order\n";
+        }; */
 
```

Comment:
There should be ` //! [placing an iceberg order]` here as well. Please see other examples.